Avoid:
Traceback (most recent call last):
  File "tools/genv8constants.py", line 97, in <module>
    match = pattern.match(line)
TypeError: cannot use a string pattern on a bytes-like object

diff -ru node-14.13.1.orig/tools/genv8constants.py node-14.13.1/tools/genv8constants.py
--- node-14.13.1.orig/tools/genv8constants.py	2020-10-13 09:35:57.347354596 +0000
+++ node-14.13.1/tools/genv8constants.py	2020-10-13 10:49:17.861640063 +0000
@@ -20,7 +20,7 @@
 outfile = open(sys.argv[1], 'w')
 try:
   pipe = subprocess.Popen([ 'objdump', '-z', '-D', sys.argv[2] ],
-      bufsize=-1, stdout=subprocess.PIPE).stdout
+      bufsize=-1, stdout=subprocess.PIPE, text=True).stdout
 except OSError as e:
   if e.errno == errno.ENOENT:
     print('''
